inputfiles/overridingTypes.jsonc

```
@@ -1414,7 +1414,8 @@
                                             "name": "algorithm",
                                             "additionalTypes": ["RsaOaepParams", "AesCtrParams", "AesCbcParams", "AesGcmParams"]
                                         }
-                                    ]
+                                    ],
+                                    "overrideType": "Promise<ArrayBuffer>"
```

```suggestion
                                    "type": {
                                      "subtype": "ArrayBuffer"
                                    }
```

I know you just copied from the existing examples, but `overrideType` should be used only as a last resort.